adcp.py

```diff
@@ -15,7 +15,7 @@ class ADCP(sensor.Sensor):
     def __init__(
             self,
             data_dir=f'/home/{config.munkholmen_user}/olmo/munkholmen/DATA',
-            file_regex_l1=r"adcp_(\d{14})\.dat",
+            file_search_l1=r"adcp_(\d{14})\.dat",
```

Comment:
should we add a comment in-line here to explain exactly what L1 data here is?

Reply:
Note that this type of thing (l0 l1.... occurs probably on 50 lines throughout the code, so probably it shouldn't sit here, but rather some other place (as if you put it here, you shoudl put it in all cases). I think better would be to take the time to remove this, and rather just have a list of such varialbes, which can have any length. But someone then has to do that work, which is a lower priority than other things for me at least.

Reply:
BTW, I would be happy is someone wants to talk through what I mean in the above with "lists", instead of l0, l1... Or wants to make a PR with a different way of handling this that they think is better - currently this system is from an old way of thinking, and there are actually a few changes with how we handle variables in the sensor classes that I'd like to change (but don't for time reasons).